src/types/zetkin.ts

```diff
@@ -0,0 +1,24 @@
+export interface ZetkinMembership {
+    organization: {
+        id: number;
+        title: string;
+    };
+    profile: {
+        id: number;
+    };
+}
+
+export interface ZetkinEventResponse {
+    action_id: number;
+    response_date: string;
+    person: {
+        name: string;
+        id: number;
+    };
+    id: number;
+}
+
+export interface ZetkinEventSignup{
```

Comment:
There is a space missing before `{`. I'm surprised the linter didn't catch this. Maybe you could add it to the configuration? See [space-before-blocks](https://eslint.org/docs/rules/space-before-blocks).

Reply:
We already use `space-before-blocks` and seems like this is currently not working on `interface` https://github.com/typescript-eslint/typescript-eslint/issues/1606